config/locales/pages/privacy_policy/en.yml

```diff
@@ -0,0 +1,145 @@
+        <li><a class='govuk-link' href='#how-we-use-itt-trainee-data'>How we use ITT trainee data</a></li>
+        <li><a class='govuk-link' href='#sharing-itt-trainee-data-within-dfe'>Sharing ITT trainee data within DfE</a></li>
+        <li><a class='govuk-link' href='#sharing-itt-trainee-data-with-third-parties'>Sharing ITT data with third parties</a></li>
+        <li><a class='govuk-link' href='#general-data-protection-regulation-gdpr'>General Data Protection Regulation (GDPR)</a></li>
+        <li><a class='govuk-link' href='#raise-a-concern'>Raise a concern</a></li>
+        <li><a class='govuk-link' href='#keep-up-to-date'>Keep up to date</a></li>
+      </ul>
+
+      <h2 class='govuk-heading-m' id='what-personal-data-we-collect-and-why'>What personal data we collect and why</h2>
+
+      <p class='govuk-body'>We (DfE) use your personal data to run and improve Register trainee teachers.</p>
+
+      <p class='govuk-body'>We collect your name and email address so that we can:</p>
+      <ul class='govuk-list govuk-list--bullet'>
+        <li>enable you to access Register trainee teachers</li>
+        <li>validate your role in your organisation</li>
+        <li>send important updates about the service</li>
+        <li>get in touch with you about any queries you raise</li>
+        <li>ask if you would like to take part in user research</li>
+        <li>record a timeline of your activity so that you can review trainee progress</li>
+        <li>create log files of your activity and any error messages you come across</li>
+      </ul>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'>If you take part in user research, you may choose to share additional information about you.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'>We use Google Analytics software to collect information about how you use Register trainee teachers. This includes IP addresses. It helps us to improve the service.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'>We do not store your personal information through Google Analytics. The data is anonymised before being used for analytics processing.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'><a href='/cookies' class='govuk-link'>Find out more about our cookie policy</a></p>
+
+      <h2 class='govuk-heading-m' id='our-lawful-basis-for-processing-your-personal-data'>Our lawful basis for processing your personal data</h2>
+
+      <p class='govuk-body'>Running and improving the service is a public task, as defined in data protection law. However, we ask for your consent to take part in user research.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'><a href='https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/' class='govuk-link'>Read about consent and public task on the Information Commissioner's Office (ICO) website</a></p>
```

Comment:
Is this margin override needed? Same q for all overrides.

Reply:
This html was taken from the prototype

Reply:
removed https://github.com/DFE-Digital/register-trainee-teachers/pull/942/commits/742b292841880f2239a6bd0eb7f99c90054e0680